cashctrl_ledger/ledger.py

```diff
@@ -133,6 +133,16 @@ def mirror_account_chart(self, target: pd.DataFrame, delete: bool = True):
         accounts = set(target_df['account']).difference(set(current_state['account']))
         to_add = target_df.loc[target_df['account'].isin(accounts)]
 
+        # Update account categories
+        def get_nodes_list(path: str) -> List[str]:
+            parts = path.strip('/').split('/')
+            return ['/' + '/'.join(parts[:i]) for i in range(1, len(parts) + 1)][1:]
```

Comment:
Clever solution to append `[1:]`!

Btw: I was not sure at first if this also works for paths consisting only of a root node, e.g. '/root-node' or '/'. 
Both work fine.
```python
>>> from typing import List
>>> def get_nodes_list(path: str) -> List[str]:
>>>     parts = path.strip('/').split('/')
>>>     return ['/' + '/'.join(parts[:i]) for i in range(1, len(parts) + 1)][1:]
>>> get_nodes_list("/root-node")
[]
>>> get_nodes_list("/")
[]

```